### PR TITLE
Dark mode.

### PR DESCRIPTION
**Pull request by kachbit**
Toggle dark mode feature accessible in the settings dialog.

### CODE REVIEW

**Comment by kachbit** [comment]
conflicts resolved ✅

**Comment by kachbit** [comment]
@abi do you plan on merging this?

**Comment by abi** [comment]
Yes probably tomorrow

**Comment by abi** [comment]
Thanks for this, @kachbit ! Would appreciate help keeping the dark mode working over time as we add new elements.

File: frontend/src/components/SettingsDialog.tsx [hunk 1]
```diff
@@ -103,53 +109,90 @@ function SettingsDialog({ settings, setSettings }: Props) {
...
+                <div className="flex items-center justify-between">
+                  <Label htmlFor="app-theme">
+                    <div>App Theme</div>
+                  </Label>
+                  <div>
+                    <button
+                      className="flex rounded-md border border-input bg-transparent px-3 py-1 text-sm shadow-sm transition-colors file:border-0 file:bg-transparent file:text-sm file:font-medium placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50t"
+                      onClick={() => {
+                        document
+                          .querySelector("div.mt-2")
```
[the remark this answers is not in the document]
Comment (abi) [reply frontend/src/components/SettingsDialog.tsx:159]: It doesn't work without it! I was curious if you knew why.

[the remark this answers is not in the document]
Comment (kachbit) [reply frontend/src/components/SettingsDialog.tsx:159]: You are referring to the "// enable dark mode for sidebar" line, correct? I just ran it locally and it worked fine after I removed the line.

**Pull request merged**